src/mount/apple.rs

```diff
@@ -0,0 +1,125 @@
+use crate::{Errno, NixPath, Result};
+use libc::c_int;
+
+libc_bitflags!(
+    /// Used with [`Nmount::nmount`].
+    pub struct MntFlags: c_int {
+        /// Do not interpret special files on the filesystem.
+        MNT_NODEV;
+        /// file system supports content protection
+        MNT_CPROTECT;
+        /// filesystem is stored locally
```

Comment:
Seems to be a copy-paste error:

```suggestion
        /// file system is quarantined
```